Setup/Patch/Data/UpdateCategoryAttribute.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Yotpo\Core\Setup\Patch\Data;
+
+use Magento\Eav\Setup\EavSetupFactory;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+
+class UpdateCategoryAttribute implements DataPatchInterface
```

Comment:
What's the reason we need the `UpdateCategoryAttribute` data patch?
Is this for customers who already installed the extension and we want to update the attribute in their store?

Reply:
Yes, you are right here